app/uk/gov/hmrc/gform/eval/ExpressionResult.scala

```diff
@@ -147,6 +160,7 @@ sealed trait ExpressionResult extends Product with Serializable {
     case t: DateResult    => false
     case t: PeriodResult  => er.ifPeriodResult(t.value.toTotalMonths >= _.toTotalMonths)
     case t: AddressResult => false
+    case t: ListResult    => er.ifNumberResult(n => t.list.exists(_.ifNumberResult(_ >= n)))
```

Comment:
Similar as cases above?

Applies for the rest of `case t: ListResult` cases

Reply:
Thanks, will do